### ISSUE TITLE
Unable to retrieve CustomSite with sfdx force:source:retrieve

### ISSUE THREAD

**Issue by mnunezdm** [forcedotcom/cli#1448]
### Summary

I think I found a bug when trying to retrieve a CustomSite using the `source:retrieve`. I'm currently running in the latest rc (7.143.0) but some colleagues are also having trouble with older releases (7.133.0, 7.138.1)

When using the mdapi:retrieve everything works fine

### Steps To Reproduce:

Run sfdx force:source:retrieve -m CustomSite

or

Run sfdx force:source:retrieve -m CustomSite:APINAME

### Expected result

Retrieve all the CustomSite or specific one

### Actual result

```shell
$ sfdx force:source:retrieve -m CustomSite                              
Preparing retrieve request... done
=== Retrieved Source
No results found

$ sfdx force:source:retrieve -m CustomSite:APINAME
Preparing retrieve request... done
=== Retrieved Source
No results found
```

### System Information

```json
{
        "cliVersion": "sfdx-cli/7.143.0",
        "architecture": "wsl-x64",
        "nodeVersion": "node-v16.13.0",
        "pluginVersions": [
                "@oclif/plugin-autocomplete 0.3.0 (core)",
                "@oclif/plugin-commands 1.3.0 (core)",
                "@oclif/plugin-help 3.3.1 (core)",
                "@oclif/plugin-not-found 1.2.6 (core)",
                "@oclif/plugin-plugins 1.10.11 (core)",
                "@oclif/plugin-update 1.5.0 (core)",
                "@oclif/plugin-warn-if-update-available 1.7.3 (core)",
                "@oclif/plugin-which 1.0.4 (core)",
                "@salesforce/analytics 1.0.2",
                "@salesforce/sfdx-plugin-lwc-test 0.1.7 (core)",
                "alias 1.2.1 (core)",
                "apex 0.11.0 (core)",
                "auth 1.8.1 (core)",
                "community 1.1.4 (core)",
                "config 1.3.23 (core)",
                "custom-metadata 1.0.12 (core)",
                "data 0.6.10 (core)",
                "generator 1.2.2 (core)",
                "info 1.3.1 (core)",
                "limits 1.3.0 (core)",
                "org 1.11.2 (core)",
                "salesforce-alm 54.0.2 (core)",
                "schema 1.1.0 (core)",
                "sfciutils 1.2.2",
                "sfdx-cli 7.143.0 (core)",
                "sfdx-codescan-plugin 1.0.7",
                "sfpowerkit 3.2.2",
                "source 1.5.3 (1.5.3)",
                "telemetry 1.4.0 (core)",
                "templates 54.2.0 (core)",
                "trust 1.1.0 (core)",
                "user 1.7.1 (core)"
        ],
        "osVersion": "Linux 5.4.72-microsoft-standard-WSL2"
}
```

**Comment by maggiben** [comment]
Hi, can you post here a demo org to repro ?

**Comment by cristiand391** [comment]
FYI: `"source 1.5.3 (1.5.3)",` is an old version (plublished 4 months ago), it seems you have installed it manually.
Run `sfdx plugins:uninstall source` to remove it, then the `fource:source:*` commands will use latest version from sfdx.

**Comment by jorgesolebur** [comment]
I am having the same issue as @mnunezdm. In a site with Experience Cloud:

```
sfdx force:source:retrieve -m CustomSite 
Preparing retrieve request... done
=== Retrieved Source
No results found
```

My details: 
```
@oclif/plugin-autocomplete 0.3.0 (core)
@oclif/plugin-commands 1.3.0 (core)
@oclif/plugin-help 3.3.1 (core)
@oclif/plugin-not-found 1.2.6 (core)
@oclif/plugin-plugins 1.10.11 (core)
@oclif/plugin-update 1.5.0 (core)
@oclif/plugin-warn-if-update-available 1.7.3 (core)
@oclif/plugin-which 1.0.4 (core)
@salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
@salesforce/sfdx-scanner 2.13.1
alias 1.2.1 (core)
apex 0.11.0 (core)  
auth 1.8.1 (core)  
community 1.1.4 (core)
config 1.3.23 (core)
custom-metadata 1.0.12 (core)
data 0.6.10 (core)
generator 1.2.2 (core)
info 1.3.1 (core)
lightning-flow-scanner 0.0.10
limits 1.3.0 (core)
org 1.11.2 (core)
salesforce-alm 54.0.2 (core)
schema 1.1.0 (core)
sfdmu 4.13.0
sfdx-cli 7.143.0 (core)
sfdx-git-delta 5.0.6
sfdx-mdt-plugin 0.5.1
shane-sfdx-plugins 4.43.0
├─ @mshanemc/sfdx-sosl 1.1.0
└─ @mshanemc/plugin-streaming 1.1.7
source 1.8.19 (core)
telemetry 1.4.0 (core)
templates 54.2.0 (core)
texei-sfdx-plugin 1.14.4
trust 1.1.0 (core)
user 1.7.1 (core)
```

**Comment by mnunezdm** [comment]
> FYI: `"source 1.5.3 (1.5.3)",` is an old version (plublished 4 months ago), it seems you have installed it manually. Run `sfdx plugins:uninstall source` to remove it, then the `fource:source:*` commands will use latest version from sfdx.

Same error after uninstalling

**Comment by WillieRuemmele** [comment]
I was able to reproduce this with the latest version. Marking as a bug.

Thank you for reporting 😄  @W-10910004@

**Comment by mnunezdm** [comment]
Bug fixed when installing the latest rc (1.46.0)

![image](https://user-images.githubusercontent.com/10410852/162285179-9cfd3c58-4460-4faa-ac44-d6568c2d11b4.png)

**Comment by shetzel** [comment]
This fix is now in the latest CLI release, 7.146.1
